.config.template.toml

```diff
@@ -1,45 +1,37 @@
 max_comment_length = { default = 500, optional = false, nmin = 10, nmax = 10000, type = "int", explanation = "max number of characters a comment can have. default is 500", example = 500, oob_error = "the max comment length should be between 10 and 10000" }
 post_lang = { default = "", optional = true, explanation = "The language you would like to translate to.", example = "es-cr" }
 min_comments = { default = 20, optional = false, nmin = 15, type = "int", explanation = "The minimum number of comments a post should have to be included. default is 20", example = 29, oob_error = "the minimum number of comments should be between 15 and 999999" }
+
 [settings]
-allow_nsfw = { optional = false, type = "bool", default = false, example = false, options = [true,
-    false,
-], explanation = "Whether to allow NSFW content, True or False" }
-theme = { optional = false, default = "dark", example = "light", options = ["dark",
-    "light",
-], explanation = "sets the Reddit theme, either LIGHT or DARK" }
-times_to_run = { optional = false, default = 1, example = 2, explanation = "used if you want to run multiple times. set to an int e.g. 4 or 29 or 1", type = "int", nmin = 1, oob_error = "It's very hard to run something less than once." }
+allow_nsfw = { optional = false, type = "bool", default = false, example = false, options = [true, false,], explanation = "Whether to allow NSFW content, True or False" }
+theme = { optional = false, default = "dark", example = "light", options = ["dark", "light",], explanation = "Sets the Reddit theme, either LIGHT or DARK" }
+times_to_run = { optional = false, default = 1, example = 2, explanation = "Used if you want to run multiple times. Set to an int e.g. 4 or 29 or 1", type = "int", nmin = 1, oob_error = "It's very hard to run something less than once." }
 opacity = { optional = false, default = 0.9, example = 0.8, explanation = "Sets the opacity of the comments when overlayed over the background", type = "float", nmin = 0, nmax = 1, oob_error = "The opacity HAS to be between 0 and 1", input_error = "The opacity HAS to be a decimal number between 0 and 1" }
-storymode = { optional = true, type = "bool", default = false, example = false, options = [true,
-    false,
-], explanation = "not yet implemented" }
+storymode = { optional = true, type = "bool", default = false, example = false, options = [true, false,], explanation = "Only read out title and post content, not yet implemented" }
 
 [settings.background]
-background_choice = { optional = true, default = "minecraft", example = "minecraft", options = ["minecraft", "gta", "rocket-league", "motor-gta", ""], explanation = "Sets the background for the video" }
+background_choice = { optional = true, default = "minecraft", example = "minecraft, gta, rocket-league, or motor-gta", options = ["minecraft", "gta", "rocket-league", "motor-gta", ""], explanation = "Sets the background for the video based on game name" }
```

Comment:
it's an example.... not an explanation

Reply:
Perhaps all the options should be displayed when the user is brought through the initial setup, just displaying the example can be confusing, especially for "background_choice". Maybe it should just be a URL instead of attached to a game?

Reply:
Great but that's not what your doing.
If you want to provide an an explanation then there is a tag for that
You are using the values incorrectly

Reply:
Change made, please re-review if you do not mind.